bin/xbps-query/xbps-query.1

```diff
@@ -277,6 +282,70 @@ expression wins.
 This expects an absolute path.
 This mode only works with repositories.
 .El
+.Sh FORMAT STRINGS
+Variables are package properties if not otherwise documented.
+See
+.Sx PROPERTIES
+section for a list of available properties.
+.Pp
+As example a format string like:
+.Bd -offset indent -literal
+{pkgname:<30} {installed_size!humanize :>10}\\n
+.Ed
+.Pp
+Would produce a list formatted like:
+.Bd -offset indent -literal
+libxbps                            304 KB
+xbps                               484 KB
+.Ed
+.Pp
+Format strings are parsed by the following EBNF:
+.Bd -literal
+<grammar>      ::= (text | escape | substitution)*
+<text>         ::= [^\\{}]+        -- literal text chunk
+<escape>       ::= "\\" [abfnrtv0] -- POSIX-like espace sequence
+                 | "\\{" | "\\}"    -- escaped "{" and "}"
+
+<substitution> ::= "{" variable ["!" conversion] [":" format] "}"
+<variable>     ::= [a-zA-Z0-9_-]
+
+<conversion>   ::= humanize | strmode
+
+-- Convert inode status information into a symbolic string
+<strmode> ::= "strmode"
+
+-- Format a number into a human readable form, the default is:`humanize .8Ki`:
+<humanize>     ::= "humanize" [space] [decimal] [width] [scale] [i]
+  <space>      ::= " "          -- Put a space between number and the suffix.
+  <decimal>    ::= "."          -- If the final result is less than 10, display
+                                   it using one digit.
+  <width>      ::= [0-9]+       -- Width of the output.
+  <scale>      ::= multiplier   -- Minimum scale multiplier and optionally
+                   [multiplier] -- Maxium scale multiplier.
+  <multiplier> ::= "B" -- byte
+                 | "K" -- kilo
+                 | "M" -- mega
+                 | "G" -- giga
+                 | "T" -- tera
+                 | "P" -- peta
+                 | "E" -- exa
+  <i>          ::= "i" -- Use IEE/IEC (and now also SI) power of two prefixes.
```

Comment:
```suggestion
  <i>          ::= "i" -- Use IEEE/IEC (and now also SI) power of two prefixes.
```

Reply:
Copied that from https://man.freebsd.org/cgi/man.cgi?query=humanize_number&apropos=0&sektion=3&manpath=FreeBSD+14.0-CURRENT&arch=default&format=html which uses `IEE/IEC`, but `IEEE` makes more sense, no idea if there is a reason.

Reply:
yeah I'm assuming they meant https://ieeexplore.ieee.org/document/9714443